hbase-endpoint/pom.xml

```diff
@@ -154,11 +154,6 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
This is for moving all the hadoop related dependencies to hadoop-2.0 profile or hadoop-3.0 profile?